README.md

```diff
@@ -79,6 +79,17 @@ The `dataflux_download_parallel` function is the most performant stand-alone dow
 
 The `dataflux_download_threaded` function allows for some amount of downlod parallelization while running within daemonic processes (e.g. a distributed ML workload leveraging [ray](https://www.ray.io/)). Daemonic processes are not permitted to spin up child processes, and thus threading must be used in these instances. Threading download performance is similar to that of multiprocessing for most use-cases, but loses out on performance as the thread/process count increases. Additionally, threading does not allow for signal interuption, so SIGINT cleanup triggers are disabled when running a threaded download.
 
+### Dataflux Download Benchmark Results
+
+These benchmarks were performed on a 48 core virtual machine on files of approximately 10kb each.
```

Comment:
`48 core virtual machine`, can we provide the specification of this machine for clarity? i.e., "n2-standard-48"?

Additionally, we might want to be crystal clear on the "core" vs "vCPU" wording here. Machines like "n2-standard-48" has 48 vCPUs (logical cores) but 24 physical cores so either we call it "vCPU" or "logical cores".